http/RemoteResource.cc

```diff
@@ -52,362 +49,182 @@
 #include "BESStopWatch.h"
 #include "BESLog.h"
 
-using namespace std;
-
 #define BES_CATALOG_ROOT_KEY "BES.Catalog.catalog.RootDirectory"
+#define REMOTE_RESOURCE_TMP_DIR_KEY "Http.RemoteResource.TmpDir"
 
 #define prolog string("RemoteResource::").append(__func__).append("() - ")
 #define MODULE HTTP_MODULE
 
+using namespace std;
+
 namespace http {
 
-RemoteResource::RemoteResource(shared_ptr<http::url> target_url, string uid, unsigned long expiredInterval)
-                               : d_remoteResourceUrl(std::move(target_url)), d_uid(std::move(uid)),
-                               d_expires_interval(expiredInterval) {
-
-    d_resourceCacheFileName.clear();
-
-    if (d_remoteResourceUrl->protocol() == FILE_PROTOCOL) {
```

Comment:
These checks need to be replaced before this code can be merged to master.

Reply:
We did...